lib/functoria/impl.mli

```diff
@@ -81,17 +84,20 @@ module Tbl : Hashtbl.S with type key = abstract
 type 'b f_dev = { f : 'a. 'a device -> 'b }
 (** The type for iterators on devices. *)
 
-type 'a f_if = cond:bool Key.value -> then_:'a -> else_:'a -> 'a
-(** The type for iterators in [if] nodes. *)
-
-type 'a f_app = f:'a -> x:'a -> 'a
-(** the type for iterators on [app] nodes. *)
-
 val with_left_most_device : Key.context -> _ t -> 'a f_dev -> 'a
 (** [with_left_most_device ctx t f] applies [f] on the left-most device in [f].
     [If] node are resolved using [ctx]. *)
 
-val map :
```

Comment:
did you remove `map` on purpose?

Reply:
Yes, it's not used outside the module now.